java-squid/src/test/java/org/sonar/java/se/checks/UnclosedResourcesCheckTest.java~SONARJAVA-1315 - Unclosed resources

```diff
@@ -1,46 +0,0 @@
-/*
```

Comment:
This has been cleaned up on master... and as such should be removed from your commit.